src/server/capacity_unit_calculator.cpp

```diff
@@ -122,21 +134,25 @@ void capacity_unit_calculator::add_multi_get_cu(int32_t status,
     }
 
     if (status == rocksdb::Status::kNotFound) {
+        _read_hotkey_collector->capture_hash_key(hash_key, 1);
```

Comment:
maybe you can move this into func `add_read_cu`
same with `add_write_cu`

Reply:
`capture_hash_key` and `add_read_cu` are two independent function. It will destroy the original semantics.